sdk/automation/azure-resourcemanager-automation/src/samples/java/com/azure/resourcemanager/automation/generated/SourceControlCreateOrUpdateSamples.java

```diff
@@ -31,7 +31,7 @@ public static void createOrUpdateASourceControl(com.azure.resourcemanager.automa
             .withSourceType(SourceType.VSO_GIT)
             .withSecurityToken(
                 new SourceControlSecurityTokenProperties()
-                    .withAccessToken("3a326f7a0dcd343ea58fee21f2fd5fb4c1234567")
+                    .withAccessToken("fakeTokenPlaceholder")
```

Comment:
Our CredScan seems stricter than that from swagger https://github.com/Azure/azure-rest-api-specs/blob/main/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/examples/sourceControl/createOrUpdateSourceControl.json#L17

LGTM. Mgmt will see later if this get caught by CredScan, if so, we will add logic to replace that automatically.

Reply:
Yes. CredScan does scan all files and reports the caught secrets.  azure-rest-api-specs repo probably doesn't include the CredScan tool. Thank you for taking a look at this.

Reply:
They've got CredScan as well, and sadly that passes :-(
https://dev.azure.com/azure-sdk//internal/_build/results?buildId=1820594&view=logs&j=ef5bc66f-4885-5416-1c1f-d4bdd6dcc526

Anyway we should be able to handle that.

Reply:
Is this generated file? Is this change going to be overridden by next automation?

Reply:
It is generated file, and it will be overwritten in next pass. I assume it will fail in CI next time, correct?

What we can do it that we can modify codegen to e.g. if see "token" in name then replace the value to "fakeTokenPlaceholder".